aiida_nanotech_empa/utils/gaussian_spin_wc_postprocess.py

```diff
@@ -0,0 +1,190 @@
+import numpy as np
+
+from PIL import Image
+
+import matplotlib.pyplot as plt
+
+
+def print_orb_energies(prop_dict, n_orb=4):
```

Comment:
Docstring is missing. I am surprised that pre-commit didn't complain.

Reply:
added